lib/extras/dec/decode.cc

```diff
@@ -24,18 +24,8 @@ constexpr size_t kMinBytes = 9;
 
 void BasenameAndExtension(const std::string& path, std::string* filename,
```

Comment:
It seems that now in all code paths *filename = path, so I think we should simplify this function to something like ```std::string GetEtension(const std::string& path)```

Reply:
good idea, done